includes/Modules/Search_Console.php

```diff
@@ -445,8 +445,8 @@ protected function parse_data_response( $method, $datapoint, $response ) {
 							);
 						}
 						if ( 'INET_DOMAIN' === $site->getType() ) {
-							$host = str_replace( array( 'http://', 'https://' ), '', $site->getIdentifier() );
-							if ( ! empty( $host ) && false !== strpos( $site_url, (string) $host ) ) {
+							$host = str_replace( array( 'http://', 'https://' ), '', trailingslashit( $site->getIdentifier() ) );
+							if ( ! empty( $host ) && false !== strpos( trailingslashit( $current_url ), $host ) ) {
```

Comment:
`$host` should be trailing-slashed within `strpos` as well, otherwise `! empty( $host )` will always be truthy.